components/schema-migrator/validate.sh

```diff
@@ -19,7 +19,7 @@ DATA_DIR="${COMPONENT_PATH}/seeds"
 IMG_NAME="compass-schema-migrator"
 NETWORK="migration-test-network"
 POSTGRES_CONTAINER="test-postgres"
-POSTGRES_VERSION="12"
+POSTGRES_VERSION="15"
```

Comment:
After removing the migration the postgres version bumping should not be needed. Also the postgres version on real env is 9.6 so there may be potential problems if the version is bumped

Reply:
The script is broken, probably from more than a month. Validate script seem to not be used in prow jobs and it fails without this change.
The reason is that provisioner image that is used to make DB export on deployment is using newest version of pg_dump utility, that produces the export that cannot be imported with older tools.